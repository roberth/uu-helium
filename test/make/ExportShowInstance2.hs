module ExportShowInstance2 where

data A = MkA Int 

