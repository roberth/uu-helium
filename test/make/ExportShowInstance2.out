Compiling make/ExportShowInstance2.hs
Compilation successful
No 'main' function defined in this module
